webClient/src/app/shared/editor-control/editor-control.service.ts

```diff
@@ -227,14 +228,40 @@ export class EditorControlService implements ZLUX.IEditor, ZLUX.IEditorMultiBuff
   }
 
   public closeFileHandler(fileContext: ProjectContext) {
+    console.log(`closeFileHandler start\n`);
     let cacheFileName = `${fileContext.model.fileName}:${fileContext.model.path}`;
     if (cacheFileName) {
       this.log.debug(`Clearing cache for`,cacheFileName);
       delete stateCache[cacheFileName];
     }
-    !fileContext.opened ? this.log.warn(`File ${fileContext.name} already closed.`) : fileContext.opened = false;
-    !fileContext.active ? this.log.warn(`File ${fileContext.name} already inactive.`) : fileContext.active = false;
+
+    /* dequeue dataset */
+    console.log(`closeFileHandler model File name is ${fileContext.model.fileName}`);
+    
+          /* Send DEQ dataset  */
+          let filePath = ['/', '\\'].indexOf(fileContext.model.path.substring(0, 1)) > -1 ? fileContext.model.path.substring(1) : fileContext.model.path;
+          const enqRequestUrl = ZoweZLUX.uriBroker.datasetEnqueueUri(filePath);   /* the ENQ URL */
+          let _observable = this.http.delete(enqRequestUrl);                        /* prepare to delete the ENQ */
+
+          console.log(`closeFileHandler delete request is for ${fileContext.model.fileName}`);
+
+          /* subscribe to the DELETE request */  
+          _observable.subscribe({
+            next: (response: any) => {
+              console.log(`closeFileHandler delete request OK`);              
+            },
+            error: (err) => {
+              // console.log(`closeFileHandler delete request FAILED, error status ${err.status}`); 
+              console.log(`closeFileHandler delete request FAILED.  Error is: `, err);             
```

Comment:
If there was an error, we'd want to use a Zowe Notification to see 1. error code 2. error message for why it failed

let pluginId = this.pluginDefinition.getBasePlugin().getIdentifier();
    // We can specify a different styleClass to theme the notification UI i.e. [...] message, 1, pluginId, "org_zowe_zlux_editor_snackbar"
    let notification = ZoweZLUX.notificationManager.createNotification(title, message, 1, pluginId);
    ZoweZLUX.notificationManager.notify(notification);

Reply:
If the Editor doesn't use a Notification for USS file deletion, you can just use a regular snackbar here too. Don't remember

Reply:
Keep in mind, if its an error, we want the duration to be longer